api/src/Controller/Security/AuthenticationTokenAction.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Security;
+
+use Symfony\Component\Routing\Annotation\Route;
+
+class AuthenticationTokenAction
+{
+    /**
+     * @Route("/authentication_token", methods={"POST"})
```

Comment:
There is no need to create a controller just to declare this route. It can be declared directly in the `config/routes/security.yml` file (for example) with the following content:
```yaml
authentication_token:
    path: /authentication_token
    methods: POST
```